larray/tests/test_excel.py

```diff
@@ -330,6 +330,7 @@ def test_excel_report_titles():
 
 
 @needs_xlwings
+@needs_pytables
```

Comment:
At which step `pytables` is required by the `ExcelReport` class ?

Reply:
because it loads example data which is stored in hdf